client/components/debug/logger.client.ts

```
@@ -49,7 +50,7 @@ class Logger {
   }
   error(message: any, error?: any) {
     this._log({ message, error }, LogType.ERROR);
-    if (typeof window !== undefined) {
+    if (window && typeof window !== undefined) {
```

Biome (2.1.2)

[error] Invalid typeof comparison.

Compare with one of the following string literals:


(lint/correctness/useValidTypeof)